content/_snippets/evmsc-disclaimer.md

```diff
@@ -0,0 +1 @@
+_The EVM compatible sidechains implementation :not_enabled: is a Proof of Concept extension to the XRP Ledger protocol and is for development purposes only. There isn't an official amendment and it is not available on the production Mainnet. The EVM compatible sidechain bridge is connected to the XRP Ledger Devnet. Do not sent transactions in Mainnet._ 
```

Comment:
Does proof of concept need to be capitalized?

I don't mind this, I might even prefer it, but is it okay to use _isn't _ and _is not in close proximity this way? "There_ **isn't** an official amendment and it **is not**...."

Do not **send** transactions in Mainnet.

Reply:
Thanks for fixing the typo. I've incorporated your style feedback.